src/config-schemas/openmrs-esm-patient-registration-schema.ts

```diff
@@ -1,26 +1,44 @@
-import { validators } from '@openmrs/esm-config';
+import { Type } from '@openmrs/esm-config';
 
 export const esmPatientRegistrationSchema = {
   personAttributeSections: {
-    arrayElements: {
-      name: { default: false, validators: [validators.isString] },
+    _type: Type.Array,
+    _elements: {
```

Comment:
The objects in `personAttribueSections` should have a top-level key `name`, alongside `personAttribues`. That seems to have been lost.

Reply:
```suggestion
    _elements: {
      name: {
        _type: Type.String,
        _default: false,
      },
```